dhis-2/dhis-support/pom.xml

```diff
@@ -22,6 +22,7 @@
     <module>dhis-support-system</module>
     <module>dhis-support-jdbc</module>
     <module>dhis-support-amqp</module>
+    <module>dhis-support-parser</module>
```

Comment:
Should we perhaps call the project `dhis-support-expression-parser` as there might be other "parsers" in the future?

Reply:
Done.